rules/windows/process_creation/proc_creation_win_pua_seatbelt.yml

```diff
@@ -0,0 +1,55 @@
+title: Seatbelt PUA Tool
+id: 38646daa-e78f-4ace-9de0-55547b2d30da
+status: experimental
+description: Detects the execution of the PUA/Recon tool Seatbelt via PE information of command line parameters
+author: Nasreddine Bencherchali
+references:
+    - https://github.com/GhostPack/Seatbelt
+    - https://www.bluetangle.dev/2022/08/fastening-seatbelt-on-threat-hunting.html
+date: 2022/10/18
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Image|endswith: '\Seatbelt.exe'
+        - OriginalFileName: 'Seatbelt.exe'
+        - Description: 'Seatbelt'
+        - CommandLine|contains:
+            # This just a list of the commands that will produce the least amount of FP in "theory"
+            # Comment out/in as needed in your environment
+            # To get the full list of commands see reference section
+            - ' DpapiMasterKeys'
+            - ' InterestingProcesses'
+            - ' InterestingFiles'
+            - ' CertificateThumbprints'
+            - ' ChromiumBookmarks'
+            - ' ChromiumHistory'
+            - ' ChromiumPresence'
+            - ' CloudCredentials'
+            - ' CredEnum'
+            - ' CredGuard'
+            - ' FirefoxHistory'
+            - ' ProcessCreationEvents'
+            #- ' RDPSessions'
+            #- ' PowerShellHistory'
```

Comment:
Some of these (like ChromiumHistory, InterestingFiles and FirefoxHistory) look potentially FP heavy for a high level rule

Reply:
As stated in the comment on the rule. Some of these could produce FP but since I don't have a concrete example, I can't reduce it to medium right now.